src/node/i18n/locales/zh-cn.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Would you be willing to add yourself as a codeowner for this file in case we add new strings in the future?

Reply:
Yes. It's my pleasure.